lib/Compilation.js

```diff
@@ -4086,7 +4086,7 @@ This prevents using hashes of each other and should be avoided.`);
 			if (!isSourceEqual(this.assets[file], source)) {
 				this.errors.push(
 					new WebpackError(
-						`Conflict: Multiple assets emit different content to the same filename ${file}`
+						`Conflict: Multiple assets emit different content to the same filename ${file}. Source File: ${source}`
```

Comment:
`source` is the content of the file.

You could include `assetInfo` in the message instead, this might give hints.